### DISCUSSION TITLE
Midi2LR Crash on Close MacOS/Intel

### DISCUSSION THREAD

**Discussion by ThomasWi7** [rsjaffe/MIDI2LR#535] [Problems]
Midi2LR works just fine, but on close the app always crashes. No matter if I close LR or shutdown the app itself.
Also it doesn't matter if I did something with the controller.

MAC OS 10.15.7 Catalina
Midi2LR 5.1.0.0
Lightroom Classic 11.
[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/7513763/MIDI2LR.log)
[MIDI2LRinfo.txt](https://github.com/rsjaffe/MIDI2LR/files/7513764/MIDI2LRinfo.txt)
0

**Comment by rsjaffe** [comment]
Do you have a crash log?

**Reply by ThomasWi7** [reply]
Sure ;-)
[MIDI2LR_2021-11-11.txt](https://github.com/rsjaffe/MIDI2LR/files/7519681/MIDI2LR_2021-11-11.txt)

**Comment by rsjaffe** [comment]
Does it still crash if you wait longer to close it? In the crash log, there were still some startup processes underway, and a crash then is hard to avoid. Let it run for a couple of minutes.

Also, is your computer connected to the internet when it crashes? MIDI2LR checks for updates on startup, and that might hang.

**Comment by ThomasWi7** [comment]
Thanks for your quick response.

Unfortunately the same behaviour. Yes internet is available all the time.

[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/7523602/MIDI2LR.log)
[MIDI2LRinfo.txt](https://github.com/rsjaffe/MIDI2LR/files/7523603/MIDI2LRinfo.txt)

[MIDI2LR_2021-11-11-2325crash.txt](https://github.com/rsjaffe/MIDI2LR/files/7523607/MIDI2LR_2021-11-11-2325crash.txt)
.

**Comment by rsjaffe** [comment]
Try the latest release 5.1.1.0. It fixed the problem on my test machine.

**Comment by ThomasWi7** [comment]
Many thanks, tried it 2-3 times, it looks very good, no more crashes.
Tomorrow I have a photo shoot, after that I will be able to try everything.
